source/Halibut/Transport/Protocol/TemporaryFileStream.cs

```diff
@@ -20,6 +21,13 @@ public void SaveTo(string filePath)
 
             AttemptToDelete(filePath);
             File.Move(path, filePath);
+
+            //Update the permission on the file to match inherited permission of the new location.
+            var fileInfo = new FileInfo(filePath);
+            var fileSecurity = fileInfo.GetAccessControl();
+            fileSecurity.SetAccessRuleProtection(false, false);
```

Comment:
Isn't the 2nd `false` preventing access rules from being inherited, which does not match what the comment says.